packages/tools/generator-pie-component/src/app/templates/component.__stories__.ts

```diff
@@ -0,0 +1,36 @@
+import type { Meta, StoryObj as Story } from '@storybook/web-components';
+import { Pie<%= componentName%>, <%= componentName%>Props } from '@justeattakeaway/pie-<%= fileName %>';
```

Comment:
Nit: I think there are quite a few places where the whitespace is inconsistent
```suggestion
import { Pie<%= componentName %>, <%= componentName %>Props } from '@justeattakeaway/pie-<%= fileName %>';
```